homeassistant/components/smartthings/climate.py

```diff
@@ -436,6 +437,7 @@ async def async_update(self) -> None:
         for mode in self._device.status.supported_ac_modes:
             if (state := AC_MODE_TO_STATE.get(mode)) is not None:
                 modes.add(state)
+                STATE_TO_AC_MODE[state] = mode
```

Comment:
Updating the global is not OK, if there are more than a single smartthings climate entity, they will now interfere with each other. The `SmartThingsAirConditioner` instance needs to keep track of if it should do "wind" or "fanOnly".